recipes/fsnviz/meta.yaml

```diff
@@ -1,29 +1,29 @@
 package:
   name: fsnviz
-  version: 0.1.0
+  version: 0.2.0
 
 source:
-  fn: FsnViz-0.1.0.tar.gz
-  url: https://pypi.python.org/packages/cb/ed/a9cd233386fab2800e24a5a5b58757c150e35aa0bb30069e44498db4b6a3/FsnViz-0.1.0.tar.gz
-  md5: 79ba80aeef411b89802e2b3e597b3c37
+  fn: 33816ab113d642f0c123e99ac5ac47359eea414a.tar.gz
+  url: https://github.com/bow/fsnviz/archive/33816ab113d642f0c123e99ac5ac47359eea414a.tar.gz
+  md5: 3096b741fe9b575a9d0292dcad508e80
 
 build:
-  skip: True # [py2k or py34]
+#  skip: True # [py2k or py3]
```

Comment:
This should be skipped for py2k and py34 (not just py3). I used the `concurrent.future` which I think is in stdlib since 3.5